al_aws.js

```diff
@@ -66,10 +65,10 @@ var getLambdaMetrics = function (functionName, metricName, statistics, callback)
         EndTime: new Date(),
         Period: 60*AWS_STATISTICS_PERIOD_MINUTES   /* 15 mins as seconds */
     };
-    return getMetricStatistics(params, statistics, callback);
+    return getMetricStatistics(params, callback);
 };
 
-var getKinesisMetrics = function (streamName, metricName, statistics, callback) {
+var getKinesisMetrics = function (streamName, metricName, callback) {
```

Comment:
I suggest to move this function to a new module, like `stats_templates.js`. So that consumers can re-use existing stats functions with collector specific resource IDs. Ideally a collector would just list preferred templates to use. For example, as CWE collector I'd like to collect stats for kinesis stream (arn) and CWE rule (arn), and get lambda invocation stats by default.

It probably worth doing same for `health_templates.js`.

Reply:
@kkuzmin that sounds good!